guides/source/active_record_validations.md

```diff
@@ -331,7 +345,7 @@ You can also pass in a custom message via the `message` option.
 
 ```ruby
 class Person < ApplicationRecord
-  validates :terms_of_service, acceptance: { message: 'must be abided' }
+  validates :terms_of_service, acceptance: { message: 'must be accepted' }
```

Comment:
I think "must be accepted" is the default and this example is trying to show how to change the default message. So something else like this?
```suggestion
  validates :terms_of_service, acceptance: { message: 'must be agreed to' }
```